src/blockcfg/mod.rs

```diff
@@ -50,6 +50,21 @@ pub fn block_0_get_slot_duration(block: &Block) -> Result<std::time::Duration, B
     }
 }
 
+pub fn block_0_get_slots_per_epoch(block: &Block) -> Result<Option<usize>, Block0Error> {
```

Comment:
Why does this API feel like Pascal to me? :)
Can we have a `GenesisBlock` newtype with these methods?